integrationTests/singleShard/block/executingMiniblocksSc_test.go

```diff
@@ -253,17 +685,19 @@ func syncBlock(t *testing.T, nodes []*integrationTests.TestProcessorNode, idxPro
 	fmt.Println(integrationTests.MakeDisplayTable(nodes))
 }
 
-func nodeJoinsGame(
+func playerJoinsGame(
 	nodes []*integrationTests.TestProcessorNode,
-	idxNode int,
+	sndAddress []byte,
```

Comment:
Might use TestWalletAccount as parameter and decrease param count by 2

Reply:
did not want to break the compatibility with node. so Node can send as well. In this case, the code is more reusable.